.github/actions/install-eprosima-dependencies-windows/action.yml

```diff
@@ -36,14 +36,22 @@ runs:
           -DINTERNAL_DEBUG=ON -Ax64 -T host=x64 -B build\fastdds Fast-DDS;
         cmake --build build\fastdds --config ${{ inputs.cmake_build_type }} --target install
 
-    - name: Install DDS Router
+    - name: Install Dev Utils
       shell: pwsh
       run: >
-        git clone https://github.com/eProsima/DDS-Router.git;
-        "ddsrouter_cmake ddsrouter_utils".Split(" ") | ForEach {
-          echo "building package: $_";
-          cmake -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}\eprosima\install `
-            -DCMAKE_PREFIX_PATH=${{ github.workspace }}\eprosima\install `
-            -Ax64 -T host=x64 -B build\$_ DDS-Router/$_;
-          cmake --build build\$_ --config ${{ inputs.cmake_build_type }} --target install;
-        }
+        git clone https://github.com/eProsima/dev-utils.git;
+        cmake `
```

Comment:
In the future we will need dds router for a node type, maybe is a good moment to leave it as a dependency